### ISSUE TITLE
Add Google Analytics

### ISSUE THREAD

**Issue by kmpaul** [NCAR/esds#87]
We have Google Analytics added to the Project Pythia portal, and I had Google Analytics on the Xdev site.  I think we should add Google Analytics to the ESDS site, too.  I'm happy to put together a PR on this since I have the GA project links already created.

**Comment by andersy005** [comment]
I believe this is being taken care of by 

https://github.com/NCAR/esds/blob/02755fc3d30af047a23ce3f9354b6adb68f89193/conf.py#L60

**Comment by kmpaul** [comment]
Oh.  I don't have access to that Google Analytics ID.

**Comment by andersy005** [comment]
I don't have access either... Presuming you want access to that account, @mgrover1 administers that account.

**Comment by kmpaul** [comment]
@mgrover1:  Can you add Anderson and me to that GA account?

**Comment by mgrover1** [comment]
I added you both - I could use some help with making sure I setup everything correctly...

**Comment by kmpaul** [comment]
I'll take a look tomorrow.

**Comment by kmpaul** [comment]
So.  Your Google Analytics hooks seem to be working.  At least, it is recording some clicks on the ESDS page.  Strangely, I tried implementing the exact same technique for enabling Google Analytics on the Pythia Foundations Book, and it is not recording any events on that page.  Very weird.

**Comment by kmpaul** [comment]
I'm going to close this issue for now since it seems to be working for the ESDS site.

**Comment by mgrover1** [comment]
Coming back to this - I don't think I had this set up properly before... I just went back and configured the site settings again, and it works now. I tested pulling up the site on my computer/mobile and it tracks in real time. I noticed the number of link clicks from the Twitter posts was not corresponding to tracked traffic... I can help setup the foundations Google tracking using the same process soon if that works.
